program/rust/src/processor/upd_price.rs

```diff
@@ -200,7 +200,7 @@ pub fn upd_price(
             price_data.prev_slot_ = price_data.agg_.pub_slot_;
             price_data.prev_price_ = price_data.agg_.price_;
             price_data.prev_conf_ = price_data.agg_.conf_;
-            price_data.prev_timestamp_ = clock.unix_timestamp;
+            price_data.prev_timestamp_ = price_data.timestamp_;
```

Comment:
wow did you catch a bug that was in the old code here?

Reply:
no this is introduced in this pr